scripts/bennu

```diff
@@ -2,7 +2,18 @@ set -x
 export LC_ALL=C
 export DEBIAN_FRONTEND=noninteractive
 ln -fs /usr/share/zoneinfo/America/Denver /etc/localtime
-apt-get install -y --no-install-recommends -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" bennu collectd ftp pv pybennu python3 python3-pip python3-setuptools python3-twisted python3-wheel socat systemd-timesyncd tcpdump tmux telnet vsftpd wget iperf3
+# force-bad-version required for bennu versioning (non-numeric)
+apt-get install -y --no-install-recommends -o Dpkg::Options::="--force-bad-version" -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" bennu collectd ftp pv pybennu python3 python3-pip python3-setuptools python3-twisted python3-wheel socat systemd-timesyncd tcpdump tmux telnet vsftpd wget iperf3 ca-certificates && apt-get clean
```

Comment:
The apt package versioning should be fixed now, see https://github.com/sandialabs/sceptre-bennu/pull/19 and https://github.com/sandialabs/sceptre-bennu/pull/18

Reply:
got it, removed the unnecessary `--force-bad-version`